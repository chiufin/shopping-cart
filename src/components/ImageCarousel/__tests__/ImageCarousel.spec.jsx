import React from 'react'
import { shallow } from 'enzyme'
import ImageCarousel from '../ImageCarousel'

const defaultProps = {
  images: [
    'test',
    'test2',
  ],
  imageAltText: 'Test Text',
}

const setup = () => {
  const component = shallow(<ImageCarousel {...defaultProps} />)
  return {
    getInstance: () => component,
    getImageCarousel: () => component.find(ImageCarousel),
  }
}

describe('ImageCarousel', () => {
  it('should render', () => {
    const component = setup()
    expect(component).toBeDefined()
    expect(component).toMatchSnapshot()
  })
})
