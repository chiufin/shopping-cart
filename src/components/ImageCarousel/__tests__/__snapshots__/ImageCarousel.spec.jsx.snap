// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ImageCarousel should render 1`] = `
Object {
  "getImageCarousel": [Function],
  "getInstance": [Function],
}
`;
